.eslintignore

```diff
@@ -1,3 +1,2 @@
 bootstrap.js
 data/bundle.js
-app.js
```

Comment:
Tweaked the ESLint rules below to support JSX so it won't choke on the React elements.